build.sbt

```diff
@@ -92,10 +93,13 @@ lazy val ui =
         "com.github.japgolly.scalajs-react" %%% "extra-ext-monocle3"           % scalaJsReactVersion,
         "edu.gemini"                        %%% "lucuma-core"                  % lucumaCoreVersion,
         "edu.gemini"                        %%% "lucuma-react-common"          % lucumaReactVersion,
+        "edu.gemini"                        %%% "lucuma-react-font-awesome"    % lucumaReactVersion,
         "edu.gemini"                        %%% "lucuma-react-semantic-ui"     % lucumaReactVersion,
         "edu.gemini"                        %%% "lucuma-react-resize-detector" % lucumaReactVersion,
-        "edu.gemini"                        %%% "lucuma-react-prime-react"     % lucumaReactVersion,
+        "edu.gemini"                        %%% "lucuma-react-tanstack-table"  % lucumaReactVersion,
         "edu.gemini"                        %%% "lucuma-react-floatingui"      % lucumaReactVersion,
+        "edu.gemini"                        %%% "lucuma-react-prime-react"     % lucumaReactVersion,
+        "edu.gemini"                        %%% "lucuma-prime-styles"          % lucumaPrimeStylesVersion,
```

Comment:
Do we actually need the theme CSS in lucuma-ui? I don't think we actually get it with this unless we also use the `lucuma-css` plugin.